.plzconfig

```diff
@@ -39,7 +39,7 @@ defaultdbgcppflags = --std=c++11 -g3 -DDEBUG -Wall -Wextra -Werror -Wno-unused-p
 ; it detects Java is installed and it can build them locally.
 javacworker = //tools/java:javac_worker
 junitrunner = //tools/java:junit_runner
-jarcattool = //tools/jarcat
+jarcattool = @//tools/jarcat
```

Comment:
Why is it a subrepo?

Reply:
it's not really; this makes it "absolute" so all subrepos use the one from this repo. Not super happy about having to do it but not sure how best to handle it otherwise.